steps/pvs_like_case_study/implementations/pvs_like_spark_cluster/start_spark_slurm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#start_spark_slurm.sh
+
+#SBATCH --nodes=3
+#  ntasks per node MUST be one, because multiple workers per node doesn't
+#  work well with slurm + spark in this script (they would need increasing
+#  ports among other things)
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=5
+#SBATCH --mem-per-cpu=8000
+#SBATCH --time=10:00:00
+
+# NOTE: This script has a lot in it, and most of it is probably unnecessary.
+# It is a heavily modified version of https://serverfault.com/a/776688
+
+unset SPARK_HOME
+CONDA_PATH=/ihme/homes/mkappel/miniconda3/condabin/conda # must be accessible within container
```

Comment:
this whole bash script is temporary, right? we wouldn't want your homes dir hardcoded

Reply:
Yes, temporary. I'm reimplementing this part in a new Spark utilities Python file.